### DISCUSSION TITLE
2FA - Brasil

### DISCUSSION THREAD

**Discussion by GugaS1lva** [community/community#52448] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Pessoal estou com muita dúvidas sobre o novo processo de 2FA do GitHub. Já faço uma segunda verificação da minha conta utilizando o GitHub App no meu aparelho celular. No entanto, todos os dias ainda recebo uma mensagem avisando para habilitar a autenticação em dois fatore. Queria poder utilizar meu próprio número de celular, mas não há Brasil disponível na lista de países.

![image](https://user-images.githubusercontent.com/92289607/230926173-6ecd5fff-8da7-402e-8f89-16090721a62e.png)

Alguém pode, por favor, me ajudar?

**Comment by oguzhan18** [comment]
I think disable it. I have the same problem and my country is Türkiye. It's ridiculous that it's not on the list.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @GugaS1lva ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
